src/features/expense/ui/DayExpenseItem/DayExpenseItem.tsx

```
@@ -0,0 +1,20 @@
+import { Expense } from '@/features/expense/model/types';
+
+interface DayExpenseProps {
```

ESLint

[error] 'DayExpenseProps' is defined but never used.

(@typescript-eslint/no-unused-vars)